sdks/python/apache_beam/transforms/userstate.py

```diff
@@ -60,6 +60,21 @@ def to_runner_api(self, context):
             element_coder_id=context.coders.get_id(self.coder)))
 
 
+class SetStateSpec(StateSpec):
+  """Specification for a user DoFn Set State cell"""
+
+  def __init__(self, name, coder):
+    assert isinstance(name, str)
```

Comment:
TypeError would be a better thing to raise here.

Reply:
I can change this. But other StateSpec are doing the same way. Probably change their assert in a separate PR.